src/pudl/transform/ferc1.py

```diff
@@ -132,40 +132,32 @@ class TableIdFerc1(enum.Enum):
     Package. But this works for now.
     """
 
-    FUEL_FERC1 = "core_ferc1__yearly_fuel"
-    PLANTS_STEAM_FERC1 = "core_ferc1__yearly_plants_steam"
-    PLANTS_HYDRO_FERC1 = "core_ferc1__yearly_plants_hydro"
-    PLANTS_SMALL_FERC1 = "core_ferc1__yearly_plants_small"
-    PLANTS_PUMPED_STORAGE_FERC1 = "core_ferc1__yearly_plants_pumped_storage"
-    PLANT_IN_SERVICE_FERC1 = "core_ferc1__yearly_plant_in_service"
-    PURCHASED_POWER_FERC1 = "core_ferc1__yearly_purchased_power"
-    TRANSMISSION_STATISTICS_FERC1 = "core_ferc1__yearly_transmission_statistics"
-    ELECTRIC_ENERGY_SOURCES_FERC1 = "core_ferc1__yearly_electric_energy_sources"
-    ELECTRIC_ENERGY_DISPOSITIONS_FERC1 = (
-        "core_ferc1__yearly_electric_energy_dispositions"
+    STEAM_PLANTS_FUEL = "core_ferc1__yearly_steam_plants_fuel_sched402"
+    STEAM_PLANTS = "core_ferc1__yearly_steam_plants_sched402"
+    HYDROELECTRIC_PLANTS = "core_ferc1__yearly_hydroelectric_plants_sched406"
+    SMALL_PLANTS = "core_ferc1__yearly_small_plants_sched410"
+    PUMPED_STORAGE_PLANTS = "core_ferc1__yearly_pumped_storage_plants_sched408"
+    PLANT_IN_SERVICE = "core_ferc1__yearly_plant_in_service_sched204"
+    PURCHASED_POWER_AND_EXCHANGES = (
+        "core_ferc1__yearly_purchased_power_and_exchanges_sched326"
```

Comment:
For both the TableId's as well as the table transformer classes, these were all previously a direct translation of the table name. Bc of the info implied by the module (the core == transform, ferc1 = ferc1, ferc1 always yearly), I decided to go with the nugget of the table name instead of the full table name. I also decided to not include the schedule id because this ID is accessible with TableId  value which itself is accessible within all of the table transformer classes.